lib/mix/tasks/phx.gen.auth.ex

```diff
@@ -363,8 +366,12 @@ defmodule Mix.Tasks.Phx.Gen.Auth do
   end
 
   defp scope_config_string(context, key, default_scope) do
+    context_app =
+      (Mix.Phoenix.in_umbrella?(File.cwd!()) && String.to_atom("#{context.context_app}_web")) ||
+        context.context_app
```

Comment:
Isn't there already a key in `context` that returns the information we need, such as `context.web_app`?